### PR TITLE
Fix sanger snvs

### LINKED ISSUE #998: Problems after update to 4.1.0 - Sanger verification
Would appreciate if the pop-up would be not according to site - but according to person that clicked on the Sanger button - We have some cases where Sanger has been verified by CMMS or Clinical immunology...
i.e. shared cases...

### PR DESCRIPTION
**Pull request by northwestwitch**
Fixes #998, namely:

- Variants with Sanger to be evaluated are queried by institute and user.
- Fixed a bug that suggested to be evaluated variants with sanger canceled.
- Showing in popup "Sanger validations to evaluate" both link to cases and all their sanger-ordered variants.

### CODE REVIEW

**Comment by northwestwitch** [comment]
I'd like to make some small changes, so please wait for the review!

**Comment by northwestwitch** [comment]
Ready to be reviewed. Some fixes had to be done but I'm also working on a generalization of the sanger validations to allow also SVs to be validated. I'll work on the other PR today, but this is good to go.

File: scout/adapter/mongo/variant_events.py [hunk 1]
```diff
@@ -201,7 +201,8 @@ def sanger_ordered_by_institute(self, institute_id):
         results = self.event_collection.aggregate([
             {'$match': {
                 '$and': [
-                    {'verb': 'sanger' },
+                    {'verb': 'sanger'},
+                    {'user_id': user_id},
```
[the remark this answers is not in the document]
Comment (northwestwitch) [reply scout/adapter/mongo/variant_events.py]: Hi, you mean institute and user don't have to be mandatory?

[the remark this answers is not in the document]
Comment (moonso) [reply scout/adapter/mongo/variant_events.py]: Yes, I've updated. See what you think. In this way the function can be used in more places even without an institute.

**Pull request merged**